src/Indice.Features.Cases.AspNetCore/Data/CasesDbContext.cs

```diff
@@ -45,8 +46,24 @@ public class CasesDbContext : DbContext
     /// <inheritdoc />
     protected override void OnModelCreating(ModelBuilder modelBuilder) {
         modelBuilder.ApplyJsonFunctions();
+        modelBuilder.HasSequence<int>("ReferenceNumberSequence", CasesApiConstants.DatabaseSchema);
+
         base.OnModelCreating(modelBuilder);
         modelBuilder.HasDefaultSchema(CasesApiConstants.DatabaseSchema);
         modelBuilder.ApplyConfigurationsFromAssembly(Assembly.GetExecutingAssembly());
     }
+
+    /// <summary>
+    /// Gets the next value of the `ReferenceNumberSequence` sequence.
+    /// </summary>
+    public async Task<int> NextReferenceNumber() {
```

Comment:
αφαιρεσε το απο το `dbContext` (δεν ειναι η δουλεια του άλλωστε αυτή) και βαλτο για την ώρα μεσα στο αντίστοιχο service, και αν μπορει να γινει κατευθειαν `NEXT VALUE FOR ReferenceNumberSequence`, χωρις database & schema.